packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -339,23 +356,20 @@ void WebView::InitWebView() {
   EwkInternalApiBinding::GetInstance().view.OffscreenRenderingEnabledSet(
       webview_instance_, true);
 
-  Ewk_Settings* settings = ewk_view_settings_get(webview_instance_);
-
   Ewk_Context* context = ewk_view_context_get(webview_instance_);
-  Ewk_Cookie_Manager* manager = ewk_context_cookie_manager_get(context);
-  if (manager) {
+  Ewk_Cookie_Manager* cookie_manager = ewk_context_cookie_manager_get(context);
+  if (cookie_manager) {
     ewk_cookie_manager_accept_policy_set(
-        manager, EWK_COOKIE_ACCEPT_POLICY_NO_THIRD_PARTY);
+        cookie_manager, EWK_COOKIE_ACCEPT_POLICY_NO_THIRD_PARTY);
   }
-  EwkInternalApiBinding::GetInstance().settings.ImePanelEnabledSet(settings,
-                                                                   true);
-  ewk_settings_javascript_enabled_set(settings, true);
```

Comment:
Is javascript enabled still true after deleting this line?

Reply:
Yes, and actually it's not deleted. `ApplySettings` is always called during the webview initialization and `ewk_settings_javascript_enabled_set` is called there.